.github/helpers/contracts/dummy-dao/deposit.js

```diff
@@ -68,7 +68,7 @@ async function deposit() {
   for (let index = 0; index < tokens.length; index++) {
     const token = tokens[index];
     // approve
-    const erc20TokenContract = new ethers.Contract(
+    const tokenTokenContract = new ethers.Contract(
```

Comment:
confusing name. maybe `tokenBasedContract` ?

Reply:
Fixed in 001e63983523604128e4d54d8ad4747026b05f2b